### DISCUSSION TITLE
essay/dev/migrate-bt-server

### DISCUSSION THREAD

**Discussion by giscus** [ATQQ/sugar-blog#315] [Announcements]
# essay/dev/migrate-bt-server

粥里有勺糖的个人博客，记录随笔与学习笔记，大前端相关的知识，高频面试题，个人面经等

https://sugarat.top/essay/dev/migrate-bt-server.html

**Comment by tenyon61** [comment]
docker用的多的话，1p很丝滑。
php项目体验没有宝塔方便。

**Comment by chenxuuu** [comment]
宝塔八年用户了，我用下来，感觉1panel更方便，毕竟所有服务都是docker
